examples/platforms/efr32/radio.c

```diff
@@ -770,15 +773,19 @@ void efr32RadioProcess(otInstance *aInstance)
     CORE_EXIT_CRITICAL();
 }
 
-void RAILCb_RfReady(void)
+void RAILCb_RxAckTimeout(void)
 {
+    if (sIsAckRequested)
```

Comment:
This check is added only for unicast packet which needs ACK from peer. `sIsAckRequested` will be set to `false` if no ACK requested (for example, transmit broadcast packet) in line 322. If ACK is received, at line 766, `sIsAckRequested` will be set to  `false`.